daprdocs/content/en/python-sdk-docs/python-actor.md

```diff
@@ -56,4 +56,68 @@ async def main():
 
 ## Sample
 
-Visit [this page](https://github.com/dapr/python-sdk/tree/release-1.0/examples/demo_actor) for a runnable actor sample.
\ No newline at end of file
+Visit [this page](https://github.com/dapr/python-sdk/tree/release-1.0/examples/demo_actor) for a runnable actor sample.
+
+
+## Mock Actor Testing
+
+The Dapr Python SDK provides the ability to create mock actors to unit test your actor methods and how they interact with the actor state.
+
+### Sample Usage 
+
+
+```
+from dapr.actor.runtime.mock_actor import create_mock_actor
+
+class MyActor(Actor, MyActorInterface):
+    async def save_state(self, data) -> None:
+        await self._state_manager.set_state('state', data)
+        await self._state_manager.save_state()
+
+mock_actor = create_mock_actor(MyActor, "id")
+
+await mock_actor.save_state(5)
+assert mockactor._state_manager._mock_state == 5 #True
+```
+Mock actors work by passing your actor class as well as the actor id (str) into the function create_mock_actor, which returns an instance of the actor with a bunch of the internal actor methods overwritten, such that instead of attempting to interact with Dapr to save state, manage timers, etc it instead only uses local variables.
```

Comment:
```suggestion
Mock actors are created by passing your actor class and an actor ID (a string) to the create_mock_actor function. This function returns an instance of the actor with many internal methods overridden. Instead of interacting with Dapr for tasks like saving state or managing timers, the mock actor uses in-memory state to simulate these behaviors.
```